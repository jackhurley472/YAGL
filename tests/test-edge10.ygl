Graph demo;

Node portland("Portland");
Node pittsburgh("Pittsburgh");
Node nyc("New York City");
Node nj("New Jersey");

demo: + portland + pittsburgh ->2566 portland ->2566 pittsburgh
      + nyc [800<->800] pittsburgh, portland [2900<->2900] nyc,
      + nj <-13 nyc <-13 nj [2899<->2899] portland,
      nj [790<->790] pittsburgh;

print(demo);
