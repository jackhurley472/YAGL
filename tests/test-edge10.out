============== Graph Print ===============
	Stats: 	4	4	4

Nodes:
Node (0) : Portland --- Node (1) : Pittsburgh --- Node (2) : New York City
Node (3) : New Jersey

Edges:
[(1) : Pittsburgh] ---(2566)---> [(0) : Portland]
[(1) : Pittsburgh] ---(800)---> [(2) : New York City]
[(1) : Pittsburgh] ---(790)---> [(3) : New Jersey]
[(0) : Portland] ---(2566)---> [(1) : Pittsburgh]
[(0) : Portland] ---(2900)---> [(2) : New York City]
[(0) : Portland] ---(2899)---> [(3) : New Jersey]
[(2) : New York City] ---(800)---> [(1) : Pittsburgh]
[(2) : New York City] ---(2900)---> [(0) : Portland]
[(2) : New York City] ---(13)---> [(3) : New Jersey]
[(3) : New Jersey] ---(13)---> [(2) : New York City]
[(3) : New Jersey] ---(2899)---> [(0) : Portland]
[(3) : New Jersey] ---(790)---> [(1) : Pittsburgh]
============ End Graph Print =============
